src/features/singleRight/delegate/ChooseServicePage/ChooseServicePage.tsx

```diff
@@ -189,6 +194,20 @@ export const ChooseServicePage = () => {
     }
   };
 
+  const onCancel = () => {
+    const cleanHostname = window.location.hostname.replace('am.ui.', '');
+    window.location.href =
+      'https://' +
+      cleanHostname +
+      '/' +
+      String(GeneralPath.Altinn2SingleRights) +
+      '?userID=' +
+      getCookie('AltinnUserId') +
+      '&amp;' +
+      'partyID=' +
+      getCookie('AltinnPartyId');
+  };
+
```

Comment:
Again I can't comment where I really want to, but since you've introduced the next page and routing to the next page, could you maybe also add routing to this next page in the CollectionBar as well?
It should take the path as a parameter if I remember correctly.

Reply:
Yup, sorry I didn't do it already. Forgot it

Reply:
No worries 😊